lib/openshift/machine.rb

```diff
@@ -15,5 +15,10 @@ def ready?(user: nil, cached: true, quiet: false)
       instance_state = raw_resource(user: user, cached: cached, quiet: quiet).dig('status','providerStatus','instanceState')
       instance_state == 'running'
     end
+
+    def machine_set(user: nil, cached: true, quiet: false)
```

Comment:
method returns a String thus better name it `machine_set_name`. To return the machine set you can do:
```
def machine_set(user: nil, cached: true, quiet: false)
  MachineSet.new(
    name: machine_set_name(user: user, cached: cached, quiet: quiet),
    project: project).tap { |m| m.default_user = default_user(user) }
  )
end
```